src/Trace/Tracer/ExtensionTracer.php

```diff
@@ -42,16 +42,30 @@ class ExtensionTracer implements TracerInterface, SpanEventHandlerInterface
      */
     private $hasSpans = false;
 
+    private $exporter;
+
+    private $spanLimit = 100;
+
     /**
      * Create a new ExtensionTracer
      *
      * @param SpanContext|null $initialContext The starting span context.
+     * @param null $exporter.
+     * @param array $options
```

Comment:
add about the span_limit/span_buffer_size here

Reply:
done